### PR TITLE
My catering bag

### PR DESCRIPTION
**Pull request by sathyaprabaS**

### CODE REVIEW

File: src/pageDrawer/CateringSelectedProductDrawer.tsx [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+function CateringSelectedProductDrawer(props: IProps) {
+  const { isOpen, handleClose, productInfo, productQuantities } = props;
+
+  return (
+    <Drawer
+      anchor="right"
+      open={isOpen}
+      sx={{
+        position: "relative",
```
Comment (praveensumathi) [inline src/pageDrawer/CateringSelectedProductDrawer.tsx:37]: use paper props. use the useThemMediaQuery

File: src/pageDrawer/CateringSelectedProductDrawer.tsx [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+        },
+        "@media (max-width: 600px)": {
+          "& .MuiDrawer-paper": {
+            width: "100vw",
+            height: "100vh",
+          },
+        },
+      }}
+    >
+      <Box>
```
Comment (praveensumathi) [inline src/pageDrawer/CateringSelectedProductDrawer.tsx:51]: use react fragment

File: src/pageDrawer/CateringSelectedProductDrawer.tsx [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+          <Typography variant="h5" sx={{ fontWeight: 700 }}>
+            Catering Request List
+          </Typography>
+          <IconButton onClick={handleClose}>
+            <CloseIcon />
+          </IconButton>
+        </Box>
+        <Divider />
+        <Container>
+          {productInfo.map((product, index) => (
```
Comment (praveensumathi) [inline src/pageDrawer/CateringSelectedProductDrawer.tsx:71]: check the length

File: src/pageDrawer/CateringSelectedProductDrawer.tsx [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+                          textOverflow: "ellipsis",
+                          whiteSpace: "nowrap",
+                        }}
+                      >
+                        {product.title}
+                      </Typography>
+                      {productQuantities.map((item) => {
+                        if (item.productId === product._id) {
+                          return (
+                            <div key={item.productId}>
```
Comment (praveensumathi) [inline src/pageDrawer/CateringSelectedProductDrawer.tsx:125]: use Box here

File: src/pageDrawer/CateringSelectedProductDrawer.tsx [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+                          fontSize: "large",
+                          fontWeight: 600,
+                          overflow: "hidden",
+                          textOverflow: "ellipsis",
+                          whiteSpace: "nowrap",
+                        }}
+                      >
+                        {product.title}
+                      </Typography>
+                      {productQuantities.map((item) => {
```
Comment (praveensumathi) [inline src/pageDrawer/CateringSelectedProductDrawer.tsx:122]: check the lenght

File: src/pageDrawer/CateringSelectedProductDrawer.tsx [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+                                        </TableCell>
+                                      </TableRow>
+                                    ))}
+                                  </TableBody>
+                                </Table>
+                              </TableContainer>
+                            </div>
+                          );
+                        }
+                        return null;
```
Comment (praveensumathi) [inline src/pageDrawer/CateringSelectedProductDrawer.tsx:171]: remove return null

File: src/pageDrawer/CateringSelectedProductDrawer.tsx (hunk 2 above)
Comment (sathyaprabaS) [reply src/pageDrawer/CateringSelectedProductDrawer.tsx:51]: remove box and used fragment

File: src/pageDrawer/CateringSelectedProductDrawer.tsx (hunk 3 above)
Comment (sathyaprabaS) [reply src/pageDrawer/CateringSelectedProductDrawer.tsx:71]: length checked

File: src/pageDrawer/CateringSelectedProductDrawer.tsx (hunk 4 above)
Comment (sathyaprabaS) [reply src/pageDrawer/CateringSelectedProductDrawer.tsx:125]: used

File: src/pageDrawer/CateringSelectedProductDrawer.tsx (hunk 6 above)
Comment (sathyaprabaS) [reply src/pageDrawer/CateringSelectedProductDrawer.tsx:171]: removed

File: src/pageDrawer/CateringSelectedProductDrawer.tsx (hunk 5 above)
Comment (sathyaprabaS) [reply src/pageDrawer/CateringSelectedProductDrawer.tsx:122]: length checked

**Review by praveensumathi — APPROVED** [review]

**Pull request merged**